docs/topics/forms/fieldsets/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Group form fields
+layout: default
+parent: Web forms
+nav_order: 5
+description: Group form fields that belong together inside a fieldset with a descriptive legend.
+---
+
+# Fieldset and legend to group form fields
+
+Group form fields that belong together inside a `<fieldset>` with a descriptive `<legend>`. The advantage of this is that the fields are  visually and in the code connected to each other. The <legend> serves as the name of the group.
+
+The `fieldset` and `legend` elements are a standard way to group fields. When a screen reader enters a fieldset, the `legend` of that group will be announced. This is a helpful way of grouping related fields together, such as differentiating between an order address and a shipping address; but is critical when grouping radio buttons and checkboxes.
+
+
+<img src="{{site.baseurl}}/assets/images/fieldset-radio-buttons.png" alt="Screenshot of grouped radio buttons that let you select the best color">
+
+
+```html
+<fieldset>
+    
+    <Legend>Which is the best color?</legend>
+
+    <input name="colorOption" type="radio" id="purple" />
+    <label for="purple">Purple</label>
+    
+    <input name="colorOption" type="radio" id="aubergine" />
+    <label for="aubergine">Aubergine</label>
+    
+    <input name="colorOption" type="radio" id="magenta" />
+    <label for="magenta">Magenta</label>
+    
+    <input name="colorOption" type="radio" id="all" />
+    <label for="all">All of the above</label>
+
+</fieldset>
+```
+
+## WCAG Succes Criteria for fieldsets
+
+Using a fieldset and legend to group of form fields is necessary to meet the WCAG success criteria:
```

Comment:
This isn't strictly accurate; WCAG requires that the structure and relationships are programmatically determinable, and fieldset/legend is one way to meet that. I'd support saying that it's the *preferred* way to meet the criteria, but using `role="group"` and `aria-labelledby` is valid when necessary.